stock_usability/models/stock_warehouse_orderpoint.py

```diff
@@ -41,6 +41,7 @@ class StockWarehouseOrderpoint(models.Model):
     @api.depends('product_id', 'location_id')
     def _compute_rotation(self):
         for rec in self:
```

Comment:
Es mejor por temas de performance y un poco más limpio hacer:
for rec in self.filtered('product_id'):